tests/Api/Parser/JsonParserTest.php

```diff
@@ -159,7 +80,7 @@ private function generateTestClient(Service $service, $args = [])
         );
     }
 
-    private function generateTestService()
+    private function generateTestService($expectedFormat)
```

Comment:
Instead of having a dynamic test service, it may be more idiomatic of the way models are constructed to just have 2 operations that have distinct response shapes, one that uses a unix timestamp member, and one that uses an iso8601 timestamp member. In the data provider above, you can then specify which operation to invoke in the test for each test case. Same would go for the XmlParserTest.

Reply:
No you were clear, I just was very tired and read it too quickly, my apologies.  I think I got what you asked for this time.